fuel-core-interfaces/src/models/mod.rs

```diff
@@ -0,0 +1,7 @@
+mod block;
```

Comment:
We don't use rust 2015 style modules in our codebases. Please convert this into a `models.rs` file in the parent dir.

Reply:
sure, will make a change